tools/build.ts

```diff
@@ -80,6 +81,51 @@ logger.info('Build.SubModules', '(2/2) Fetching submodules... DONE');
 
 logger.info('Build', 'Updating submodules... DONE');
 
+logger.info('Build', 'Preparing test data...');
+const prepareTestData = cleanInstall || !fs.existsSync(TEST_DATA_NODE);
+if (prepareTestData) {
+  // Step 1: Clean folders if needed
+  logger.info('Build.TestData', '(1/3) Cleaning folder...');
+  if (cleanInstall) {
+    fs.emptyDirSync(TEST_DATA_NODE);
+  }
+  logger.info('Build.TestData', `(1/3) Cleaning folder... ${cleanInstall ? 'DONE' : 'SKIPPED'}`);
+  // Step 2: copy node tests for different version
+  logger.info('Build.TestData', '(2/3) Copy tests...');
+  [['v7', '5af210ca8a1c73aa6bae8754c9346ec54d0a756e'],   // rel-1.2.3
+   ['v8', '5cc146270945f0d007b140fb59a892a60ba69f49'],   // rel-1.3.0
+   ['v9', '4e67414849122d3df78bd72cee5717f90e715d12'],   // rel-1.4.1
+   ['v10', 'b22041c3f16cf8bcca9ed93982d6ffdf6ebf3746'],  // master
```

Comment:
Is the general idea to freeze a commit id after every onnx release ? For example after onnx locks down v10, we freeze the commit id for that and we keep rolling forward the commit id of master (for v11, and so on) ?

Reply:
yes that is the idea.